tests/suite/cli_rustup_init_ui.rs

```diff
@@ -0,0 +1,65 @@
+use std::fs::create_dir_all;
+use std::path::Path;
+
+use snapbox::cmd::cargo_bin;
+use snapbox::cmd::Command;
+use snapbox::Data;
+
+#[track_caller]
+fn test_help(name: &str, args: &[&str]) {
+    let home = Path::new(env!("CARGO_TARGET_TMPDIR")).join("home-ro");
+    create_dir_all(&home).unwrap();
+
+    let rustup_init = cargo_bin!("rustup-init");
+    Command::new(rustup_init)
+        .env("RUSTUP_TERM_COLOR", "always")
```

Comment:
I went with `RUSTUP_TERM_COLOR=always`  and `term.svg` tests in prep for adding color to clap's help and errors (blocked on #4520).